### ISSUE TITLE
3.2.4 - es5 error

### ISSUE THREAD

**Issue by yevgenypats** [mozilla-frontend-infra/react-lazylog#34]
Looks a similar problem to [https://github.com/mozilla-frontend-infra/react-lazylog/issues/11](https://github.com/mozilla-frontend-infra/react-lazylog/issues/11)

The error was fixed in 3.1.4 using the css import but now it is back in 3.2.4 with a bit different error.

Uncaught TypeError: r.n is not a function

I guess this also somekind of API change.

Cheers

**Comment by helfi92** [comment]
Hi @yevgenypats,

I tried it using

```jsx
import LazyLog from 'react-lazylog/es5/LazyLog';
import 'react-lazylog/es5/index.css';

// ...

<LazyLog url="https://gist.githubusercontent.com/helfi92/96d4444aa0ed46c5f9060a789d316100/raw/ba0d30a9877ea5cc23c7afcd44505dbc2bab1538/typical-live_backing.log" />
```

but that worked for me. If you could produce a reduced test repo, that would be really helpful. Feel free to use something like https://codepen.io/ if that's easier for you.

> The error was fixed in 3.1.4 using the css import but now it is back in 3.2.4 with a bit different error.

The latest release is `3.2.1`. `3.2.4` shouldn't exist. https://github.com/mozilla-frontend-infra/react-lazylog/releases.

**Comment by helfi92** [comment]
Give `v4.0.0` a try. It's a breaking change but will hopefully fix the issue.

Breaking change:

```js
// import { LazyLog } from 'react-lazylog/es5';
// import 'react-lazylog/es5/index.css';
import { LazyLog } from 'react-lazylog';
```
